Tools/cases_generator/tier2_abstract_generator.py

```diff
@@ -206,7 +206,7 @@ def _write_body_abstract_interp_pure_uop(
         mangled_uop.stack.inputs
     )
 
-    if uop.name == "_NOP":
+    if uop.name in {"_NOP", "_SWAP", "_POP_TOP"}:
```

Comment:
Maybe these can be recognized more systematically? (And I wonder if `_COPY` is also automatable?

Reply:
Not yet, I tried. Because we don't leave its assignment to the code generator.